extensions/2.0/Khronos/KHR_lights_dome/README.md

```diff
@@ -0,0 +1,129 @@
+# KHR_lights_dome
+
+## Contributors
+
+* Ben Houston, Threekit, <https://twitter.com/BenHouston3D>
+* Romain Guy, Google, <https://twitter.com/romainguy>
+
+## Status
+
+Proposed.
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension provides the ability to define a single image-based dome light in a glTF scene.  A dome light illuminate the scene as if the light was coming from a sphere outside of the scene extents.  Only one dome light per scene is supported to avoid confusing this dome light extension with being a specular or diffuse light probe solution.
+
+Many 3D tools and engines support image-based global illumination but the exact technique and data formats employed vary.
+
+### Differences from EXT_lights_image_based
+
+This extension is similar to the EXT_lights_image_based extension but differs in a bunch of important ways.  These differences are motivated by trying to not presuppose the implementation of the rendering method, rather than letting that leak into the data transfer format specification.  It also simplifies what is stored to the minimum, any additional data can be derived as necessary by the rendering engine.
+
+The differences are:
+
+- The cube map is stored in non-prefiltered form (in nits units), rather than prefiltered (in lux units.)
+- Only the top level full resolution cube map is stored, rather than the mip chain.
+- The cube map data, if specified via a PNG, is assumed to be encoded in R11G11B10F, rather than RGBD.
```

Comment:
I think we should consider using KTX for textures, I also think this is one of the major concerns regarding the existing EXT_lights_image_based (that it uses PNG and not KTX)

Reply:
Good idea.  I asked that on twitter here: https://twitter.com/BenHouston3D/status/1368956654776164364. I didn't get a response as to whether it is guar rented to be lossless.  Remember if one encodes R11G11B10F into an image it better be lossless or you'll get garbage out as the lossy encoder won't be error tracking properly.  Also KTX/Basis in glTF doesn't yet support a true HDR texture right?

Reply:
KTX v2 container as it natively supports cubemaps. For the payload format I'd strongly suggest BC6H as it has much better precision and lower memory footprint than packed floats.

Platforms that don't support BC6H natively, can decode it to one of three uncompressed floating-point formats with these highly-optimized wasm decoders: https://github.com/KhronosGroup/BC6H-Decoder-WASM

Reply:
I could make this extension dependent upon KTX2 + BC6H support.  Is BC6H always supported if KTX2 is supported by glTF?  Do the run-time glTF engines already support BC6H?

Reply:
BC6H support is a hardware feature.

Reply:
There appears to be no support for BC6H on mobile at this time: https://docs.unity3d.com/Manual/class-TextureImporterOverride.html.   This is likely to be the main users of glTF.  The BC6H encoder to R11G11B10F seems to be about 10 KB in size.  That is small enough to be acceptable.  This means that we could support a true native HDR format.

Reply:
To further reduce transmission size, BC6H data could be RDO-encoded (TBD) and losslessly compressed with Zstandard (already included in KTX v2, currently used only with UASTC codec as a part of `KHR_texture_basisu`).

Another work is in progress to improve mobile support.

Reply:
@lexaknyazev do you have a slide deck that covers the benefits and downsides of these formats?  I heard there is one.

Reply:
It was an internal presentation I did over a year ago and it's quite outdated now.

Here are some slides from it (updated a bit):
![image](https://user-images.githubusercontent.com/3479527/110803041-39b90a00-8298-11eb-9cf7-b4b4df52c7d1.png)
![image](https://user-images.githubusercontent.com/3479527/110803095-48072600-8298-11eb-8307-a71e5bc58ecd.png)
![image](https://user-images.githubusercontent.com/3479527/110803121-4ccbda00-8298-11eb-82a6-ddf1acc96caa.png)
![image](https://user-images.githubusercontent.com/3479527/110803252-6f5df300-8298-11eb-8be4-c2e8a0665748.png)
![image](https://user-images.githubusercontent.com/3479527/110803294-7a188800-8298-11eb-99cb-c58b784bd12c.png)
![image](https://user-images.githubusercontent.com/3479527/110803307-7dac0f00-8298-11eb-9d50-049023139462.png)
![image](https://user-images.githubusercontent.com/3479527/110803571-c237aa80-8298-11eb-8577-ce8278a62a0b.png)